engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -515,7 +515,8 @@ private void addFinalPostProcessingNodes(RenderGraph renderGraph) {
         Node initialPostProcessingNode = renderGraph.findNode("engine:initialPostProcessingNode");
         Node updateExposureNode = renderGraph.findNode("engine:updateExposureNode");
 
-        Node toneMappingNode = new ToneMappingNode("toneMappingNode", context);
+        ToneMappingNode toneMappingNode = new ToneMappingNode("toneMappingNode", context);
```

Comment:
does it make a perf difference if we create as interface type only and cast later to get to new methods ?